docs/lang/articles/contribution/style_guide_en.md

```diff
@@ -0,0 +1,374 @@
+---
+sidebar_position: 12
+---
+
+# Taichi Technical Documentation Style Guide
```

Comment:
Is this file expected to be included in this PR? The PR title is "restructure dev_install.md".

Reply:
Nice catch! I'll change the PR title accordingly.